tinynn/graph/quantization/quantizer.py

```diff
@@ -2008,3 +2095,20 @@ def load_processed_extra_qat_rules():
             base_rule_pair[0] = True
         processed_extra_qat_rules.update(rule_dict)
     return processed_extra_qat_rules
+
+
+def load_processed_rewrite_to_fuse_rules():
```

Comment:
Consider refactoring the functions. As can be seen that Line 2071-2079, 2087- 2097 and 2104-2114 are almost the same.